src/ck-manager.c

```diff
@@ -1557,6 +1557,8 @@ do_restart (CkManager             *manager,
                 return;
         }
 
+        system("run-parts /etc/Consolekit/preshutdown.d");
```

Comment:
That should probably use SYSCONFDIR.
Also the directory name should be ConsoleKit (capital K).